models/issue_comment.go

```diff
@@ -708,6 +708,12 @@ func createDeadlineComment(e *xorm.Session, doer *User, issue *Issue, newDeadlin
 		content = newDeadlineUnix.Format("2006-01-02") + "|" + issue.DeadlineUnix.Format("2006-01-02")
 	}
 
+	if issue.Repo == nil {
```

Comment:
This line is unnecessary since LoadRepo has done that.

Reply:
Changes have been made so is possible to merge this PR ?